core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixWebView.java

```diff
@@ -139,6 +139,7 @@ protected void onCreateContextMenu(ContextMenu menu) {
   @Override
   protected void onScrollChanged(int l, int t, int oldl, int oldt) {
     super.onScrollChanged(l, t, oldl, oldt);
+    System.out.println("scrooooollling");
```

Comment:
this won't work for printing logs if memory serves and should also be deleted

Reply:
more oops.. I thought I had checked what I committed before doing so, apparently not!